src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -136,16 +188,32 @@
                 } else {
                     return [];
                 }
+            },
+            uploadDisabled(){
+                if (this.choiceUpload === 'uploadFiles' && this.uploadFilesToAdr.length < 1){
+                    return true
+                } else {
+                    return false
+                }
             }
         },
         components: {
-            Modal
+            Modal,
+            HelpCircleIcon,
         },
         watch: {
             uploadFiles() {
                 this.setDefaultCheckedItems()
+            },
+            choiceUpload(){
+                if (this.choiceUpload === 'createRelease'){
+                    this.setDefaultCheckedItems()
+                }
```

Comment:
This makes sure all files are selected when toggling back to createRelease. This means, if you had unselected any files in uploadFiles and then toggled back to createRelease, those selections will be lost.